app/services/travel_to_work_area_and_london_borough_service.rb

```diff
@@ -0,0 +1,18 @@
+class TravelToWorkAreaAndLondonBoroughService
+  def self.add_travel_to_work_area_and_london_borough(site:)
+    url = URI("https://mapit.mysociety.org/point/4326/#{site.longitude},#{site.latitude}?type=TTW,LBO&api_key=#{Settings.mapit_api_key}")
```

Comment:
The host could also be added to `config/settings`

Reply:
Cool have done 👍